src/tools/mercurial/tracking_back_string_to_bug.md

```diff
@@ -1,33 +1,47 @@
-# How to identify the bug that introduced a string
+# How to identify Bugzilla bugs that introduced a string
 
-Sometimes it’s helpful to identify which bug introduced a specific string, for example to check if an issue was already reported, or find out more information on a feature.
+Sometimes it’s helpful to identify which bug on [Bugzilla](https://bugzilla.mozilla.org) introduced a specific string, for example to check if an issue was already reported, or find out more information on a feature.
+
+Note: These instructions apply only to projects that make use of Bugzilla, which currenly are Firefox projects for desktop and Android.
 
 ## Find the string
 
-The first step is to identify where the string is within the Mercurial repository, and the fastest way is to use [Transvision](https://transvision.mozfr.org).
+The first step is to identify where the string is within the Mercurial repository by using [Searchfox](https://searchfox.org/).
+
+For example, if you want to know which bug introduced the string `Password quality meter`, you can search for the [string](https://searchfox.org/mozilla-central/search?q=Password+quality+meter&path=&case=false&regexp=false). This may return multiples strings with the same text, so to find the exact string you can use the the [identifier](https://searchfox.org/mozilla-central/search?q=password-quality-meter&path=&case=false&regexp=false) (which can be found from [CONTEXT](../pontoon/ui.md#context) in Pontoon).
+
+The results may look something like this:
+
+![Searchfox](../../assets/images/mercurial/searchfox.png)
 
-For example, if you want to know which bug introduced the string `Would you like to let this site start a server accessible to nearby devices and people?`, you can search for the [string](https://transvision.mozfr.org/?recherche=Would+you+like+to+let+this+site+start+a+server+accessible+to+nearby+devices+and+people%3F&repo=gecko_strings&sourcelocale=en-US&locale=it&search_type=strings_entities&perfect_match=perfect_match) or the identifier (if you already know it).
+We can further refine results to only localization files by using `locales/en-US` in the `Path filter` field in the top right to limit search results to localization files.
```

Comment:
[We ⇾ You](https://github.com/mozilla-l10n/documentation/blob/main/src/misc/documentation_styleguide.md#general-rules)

`locales-en-US` only works for desktop, not Android. Maybe more useful to call out::
* For Firefox, limit the search to `.ftl` or `.properties` (info on the file available in CONTEXT).
* For Android, limit the search to `.xml`.

Reply:
Updated, also added a way to copy/paste check for both (`{.ftl,.properties}`)